corehq/apps/hqmedia/tasks.py

```diff
@@ -183,7 +185,9 @@ def build_application_zip(include_multimedia_files, include_index_files, app,
                 with zipfile.ZipFile(tmp, "r") as z:
                     media_suites = [f for f in z.namelist() if re.search(r'\bmedia_suite.xml\b', f)]
                     if len(media_suites) != 1:
-                        errors.append(_('Could not identify media_suite.xml in CCZ'))
+                        message = _('Could not identify media_suite.xml in CCZ')
+                        errors.append(message)
+                        notify_exception(None, "[ICDS-291] ".format(message))
```

Comment:
`"[ICDS-291] ".format(message)` will always result in `"[ICDS-291] "`.

Did you want `"[ICDS-291] {}".format(message)`

Reply:
That's embarrassing. Thank you!